Rubberduck.VBEEditor/ComManagement/IComSafe.cs

```diff
@@ -7,5 +7,8 @@ public interface IComSafe: IDisposable
     {
         void Add(ISafeComWrapper comWrapper);
         bool TryRemove(ISafeComWrapper comWrapper);
+#if DEBUG
+        void Serialize(string targetDirectory);
```

Comment:
I'm not entirely thrilled with the idea of a conditionally compiled interface, especially using DEBUG (I generally always have that flag set unless I'm running tests right before a PR, and I sometimes forget to do that).  

If we're using the DEBUG flag here, it needs XML doc warning not to use it outside of a DEBUG directive, and concrete implementations should inherit that doc. That should probably be a general rule for conditionally compiled interfaces.